src/components/M3NavigationBar/m3NavigationBar.module.scss

```
@@ -0,0 +1,124 @@
+    height: 24px;
+    align-items: center;
+    justify-content: center;
+    color: var(--m3-on-surface-variant, #444748);
+
+    > svg {
+        display: block;
+        width: 24px;
+        height: 24px;
+    }
+
+    :global {
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)